### ISSUE TITLE
All bullshit.

### ISSUE THREAD

**Issue by Exdner** [DIGITALCRIMINAL/OnlyFans#379]
This is all bullshit.Doesn't even work.I've wasted my time too much on this.No matter how manny times i check if i entered all the info correctly still gives me auth attempt shit.You don't even have a proper tutorial on this.I'm just gonna use onlyfans enhancer that actually gives me a fucking download button for me to press and that's it.

**Comment by DIGITALCRIMINAL** [comment]
Do you have 0 IQ points? The tutorial is straightforward lmao.

The only way I can help you is if you send me your cookies or log in details, but that's it. I can't do anything more, and you'll have to use something else.

**Comment by Exdner** [comment]
You're right.I overreacted the situation.I'd like to think that i have more than 0 IQ  but  saying what i said proves otherwise.
I'd like to apologize for my behaviour because clearly you're the expert here and i'm nowhere near your level.It was one of my weakest moments and i regret it.

Now, as i said i'm not a professional but i do know a thing or two.

I followed the tutorial step by step really carefully making sure i didn't make any mistakes.You also said in the other thread that i should ignore the extra_auth file , i did that too.No success.Could be something from my end but i don' think so, i'm using python 3.8 so i should be fine.
I really don't know what i could be doing wrong or what i should try anymore.Maybe i'm too dumb for this after all and i should let it go.Sorry for wasting your time.
